portal-web/test/functional/com/liferay/portalweb/macros/CompanyLogging.macro

```diff
@@ -87,7 +87,7 @@ definition {
 		return ${filePath};
 	}
 
-	@summary = "Default summary"
+	@summary = "This will get the Company ${logName}"
```

Comment:
Since logName is the returned value I think it's find to just have it as
 "This will get the company log name for company ${portalInstanceName}